auth-web/src/schemas/address-schema.ts

```diff
@@ -0,0 +1,30 @@
+import { maxLength, required } from 'vuelidate/lib/validators'
+
+// The Address schema containing Vuelidate rules.
+// NB: This should match the subject JSON schema.
```

Comment:
what is subject JSON schema??

Reply:
comes up while copying from entities code, will remove that

Reply:
thanks